BitFaster.Caching.UnitTests/DurationTests.cs

```diff
@@ -19,5 +30,20 @@ public void RoundTripDays()
             var d = Duration.FromDays(2);
             d.ToTimeSpan().Should().BeCloseTo(TimeSpan.FromDays(2), TimeSpan.FromMilliseconds(100));
         }
+
+        [Fact]
+        public void OutputTimeParameters()
```

Comment:
Stopwatch.Frequency 1000000000
TimeSpan.TicksPerSecond 10000000
stopwatchAdjustmentFactor 100
clock time 1716312996333
node time 1785032473069
adv time 1718460479980

Reply:
The list of failing tests is:

BitFaster.Caching.UnitTests.Lru.TLruTickCount64PolicyTests.WhenItemIsNotExpiredShouldDiscardIsFalse
BitFaster.Caching.UnitTests.Lru.AfterAccessPolicyTests.CreateItemInitializesTimestampToNow
BitFaster.Caching.UnitTests.Lfu.TimerWheelTests.WhenAdvanceThrowsCurrentTimeIsNotAdvanced
BitFaster.Caching.UnitTests.Lru.TLruTickCount64PolicyTests.CreateItemInitializesTimestampToNow
BitFaster.Caching.UnitTests.Lru.AfterAccessPolicyTests.WhenItemIsNotExpiredShouldDiscardIsFalse
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.RouteWarm(wasAccessed: False, isExpired: True, expectedDestination: Remove)
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.RouteHot(wasAccessed: True, isExpired: True, expectedDestination: Remove)
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.RouteHot(wasAccessed: False, isExpired: True, expectedDestination: Remove)
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.RouteWarm(wasAccessed: True, isExpired: True, expectedDestination: Remove)
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.WhenItemIsExpiredShouldDiscardIsTrue
BitFaster.Caching.UnitTests.Lru.DiscretePolicyTests.RouteCold(wasAccessed: True, isExpired: True, expectedDestination: Remove)